spec/mongo/collection_spec.rb

```diff
@@ -2000,6 +2000,39 @@ def generate
   end
 
   describe '#count_documents' do
+
+    # populate documents to add to collection
+    let(:documents) do
+      documents = []
+      1.upto(10) do |index|
+        documents << { key: 'a', _id: "in#{index}" }
+      end
+      documents
+    end
```

Comment:
I would move this into the `when collection is not empty` context since you only need it there.